### PR TITLE
Import-DB parameter override fix

### PR DESCRIPTION
**Pull request by iulianpascalau**
- removed unnecessary import-db parameter override
- simplified checkpointHashesHolder.go implementation

### CODE REVIEW

File: data/trie/hashesHolder/checkpointHashesHolder.go [hunk 1]
```diff
@@ -77,16 +80,19 @@ func (c *checkpointHashesHolder) RemoveCommitted(lastCommittedRootHash []byte) {
...
 	sizeOfRemovedHashes := uint64(0)
-	for index, hashesMap := range c.hashes {
-		sizeOfRemovedHashes = sizeOfRemovedHashes + getMapSize(hashesMap, c.hashSize)
+	for index, rootHash := range c.rootHashes {
+		mapHashes := c.hashes[index]
+		sizeOfRemovedHashes = sizeOfRemovedHashes + getMapSize(mapHashes, c.hashSize) + uint64(len(rootHash))
 
-		_, ok := hashesMap[string(lastCommittedRootHash)]
-		if !ok {
+		lastCommittedRootHashNotFound := !bytes.Equal(c.rootHashes[index], lastCommittedRootHash)
```
Comment (SebastianMarian) [inline data/trie/hashesHolder/checkpointHashesHolder.go:87]: Why not directly rootHash instead c.rootHashes[index]

File: data/trie/hashesHolder/checkpointHashesHolder.go [hunk 2]
```diff
@@ -120,45 +126,36 @@ func (c *checkpointHashesHolder) Remove(hash []byte) {
...
-func getMapSize(hashesMap map[string]data.ModifiedHashes, hashSize uint64) uint64 {
-	mapSize := uint64(0)
-	for key, values := range hashesMap {
-		keySize := uint64(len(key))
-		hashesSize := uint64(len(values)) * hashSize
-		mapSize = mapSize + keySize + hashesSize
-	}
-
-	return mapSize
+//func isInMap(hash []byte, hashesMap map[string]data.ModifiedHashes) bool {
```
Comment (SebastianMarian) [inline data/trie/hashesHolder/checkpointHashesHolder.go:146]: Remove commented code

File: data/trie/hashesHolder/checkpointHashesHolder.go [hunk 3]
```diff
@@ -120,45 +126,36 @@ func (c *checkpointHashesHolder) Remove(hash []byte) {
...
+	delete(hashesMap, string(hash))
+
+	ok = checkCorrectSize(c.currentSize, c.hashSize)
+	if !ok {
+		c.computeCurrentSize()
+		return
 	}
 
-	return false
+	c.currentSize = c.currentSize - c.hashSize
```
Comment (SebastianMarian) [inline data/trie/hashesHolder/checkpointHashesHolder.go:143]: c.currentSize -= c.hashSize

File: data/trie/hashesHolder/checkpointHashesHolder.go (hunk 1 above)
Comment (iulianpascalau) [reply data/trie/hashesHolder/checkpointHashesHolder.go:87]: fixed

File: data/trie/hashesHolder/checkpointHashesHolder.go (hunk 2 above)
Comment (iulianpascalau) [reply data/trie/hashesHolder/checkpointHashesHolder.go:146]: removed

File: data/trie/hashesHolder/checkpointHashesHolder.go (hunk 3 above)
Comment (iulianpascalau) [reply data/trie/hashesHolder/checkpointHashesHolder.go]: done

**Pull request merged**